main/reference/zoe-api/zoe.md

```diff
@@ -3,300 +3,333 @@
 
 ```js
 { externalMint: myExternalMint }
 ```
 
-It returns a promise for a `StartInstanceResult` object. The object consists of:
-- `adminFacet` `{AdminFacet}`
-- `creatorFacet` `{any}`
-- `publicFacet` `{any}`
-- `instance` `{Instance}`
-- `creatorInvitation` `{Payment | undefined}`
+It returns a **Promise** for a **StartInstanceResult** object. The object consists of:
 
-The `adminFacet` has one method:
-- `getVatShutdownPromise()`    
-  - Returns a promise that resolves to reason (the value passed to `fail(reason)`) or 
-    completion (the value passed to `exit(completion)`) when this newly started instance terminates. 
+- **adminFacet** **AdminFacet**
+- **creatorFacet** **any**
+- **publicFacet** **any**
+- **instance** **Instance**
+- **creatorInvitation** **Payment | undefined**
 
-A `publicFacet` is an object available via Zoe to anyone knowing
-the instance they are associated with. The `publicFacet` is used for general queries
+The **adminFacet** has one method:
+- **getVatShutdownPromise()**    
+  - Returns a promise that resolves to reason (the value passed to **fail(reason)**) or 
+    completion (the value passed to **exit(completion)**) when this newly started instance terminates. 
+
+A **publicFacet** is an object available via Zoe to anyone knowing
+the instance they are associated with. The **publicFacet** is used for general queries
 and actions, such as getting a current price or creating public invitations. Since a
 facet is defined just as any other object,
 the contract developer can add methods to them just like they would any object.
 
-The `creatorFacet` is only available in this return value (i.e. only when starting
+The **creatorFacet** is only available in this return value (i.e. only when starting
 a contract instance). The contract designer
 should use it to encapsulate things that the contract runner might not want to share,
 or might want to control the distribution of. The party who starts the contract
-should carefully consider the impact before sharing access to the `creatorFacet`.
+should carefully consider the impact before sharing access to the **creatorFacet**.
 
-`creatorInvitation` is an invitation that the contract instance creator can use.
+**creatorInvitation** is an invitation that the contract instance creator can use.
 It is usually used in contracts where the creator immediately sells
 something (auctions, swaps, etc.), so it's helpful for the creator to have
 an invitation to escrow and sell goods. Remember that Zoe invitations are
-represented as a `Payment`.
+represented as a **Payment**.
 ```js
 const issuerKeywordRecord = {
-  'Asset': moolaIssuer,
-  'Price': quatlooIssuer
+  Asset: moolaIssuer,
+  Price: quatlooIssuer
 };
 const terms = { numBids: 3 };
-const { creatorFacet, publicFacet, creatorInvitation } = await E(zoe).startInstance(
+const { creatorFacet, publicFacet, creatorInvitation } = await E(Zoe).startInstance(
```

Comment:
The order & set of values being returned in this example don't seem to match the documentation above

Reply:
Yup. So I've added properties to the const object that's receiving the returned StartInstanceResult so it matched the documentation. Usually I would at the very least test the code snippet rather than just blindly copy/pasting terms, but... I don't have a test harness set up at the moment, and I really don't want to spend a lot of time on code snippets at this time.